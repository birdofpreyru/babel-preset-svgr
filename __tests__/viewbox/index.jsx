/**
 * Tests the preset used without any options.
 */

import React from 'react';
import ReactDOM from 'react-dom/server';

import Logo1 from './logo-1.svg';
import Logo2 from './logo-2.svg';

test('SVG import works as expected', () => {
  let code = ReactDOM.renderToString(<Logo1 />);
  expect(code).toMatchSnapshot();
  code = ReactDOM.renderToString(<Logo2 />);
  expect(code).toMatchSnapshot();
});
