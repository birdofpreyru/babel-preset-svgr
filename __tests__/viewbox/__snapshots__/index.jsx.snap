// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVG import works as expected 1`] = `"<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"40\\" height=\\"50\\" viewBox=\\"0 0 40 50\\" data-reactroot=\\"\\"><g stroke-width=\\"0.997\\"><path fill=\\"#f5f5f5\\" d=\\"M0 0h39.993v50H0z\\"></path><g aria-label=\\"P\\" style=\\"line-height:120%;-inkscape-font-specification:Oswald\\" font-weight=\\"400\\" font-size=\\"99.751\\" font-family=\\"Oswald\\" letter-spacing=\\"0\\" word-spacing=\\"0\\" fill=\\"#313131\\" stroke-width=\\"0.397\\"><path d=\\"M14.18 34.54h5.917c4.3 0 7.58-1.216 9.84-3.648 2.261-2.433 3.391-5.838 3.391-10.216 0-4.865-1.064-8.557-3.191-11.078-2.084-2.521-5.452-3.781-10.107-3.781H6.666L6.63 39.902H0V0h39.994v39.902H14.173z\\"></path><path d=\\"M17.637 29.167c2.216 0 3.923-.243 5.12-.73 1.24-.486 2.127-1.326 2.659-2.52.532-1.195.798-2.92.798-5.175 0-2.742-.2-4.754-.599-6.036-.398-1.283-1.196-2.19-2.393-2.72-1.153-.53-2.992-.796-5.519-.796H14.18v17.977z\\"></path></g><text style=\\"line-height:120%;-inkscape-font-specification:Oswald\\" x=\\"43.138\\" y=\\"136.474\\" font-weight=\\"400\\" font-size=\\"6.063\\" font-family=\\"Oswald\\" letter-spacing=\\"0\\" word-spacing=\\"0\\" fill=\\"#313131\\" stroke-width=\\"0.264\\" transform=\\"matrix(1.00426 0 0 1.00194 -43.706 -87.826)\\"><tspan x=\\"43.138\\" y=\\"136.474\\">Dr. Pogodin Studio</tspan></text></g></svg>"`;

exports[`SVG import works as expected 2`] = `"<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"40\\" height=\\"50\\" data-reactroot=\\"\\"><g stroke-width=\\"0.997\\"><path fill=\\"#f5f5f5\\" d=\\"M0 0h39.993v50H0z\\"></path><g aria-label=\\"P\\" style=\\"line-height:120%;-inkscape-font-specification:Oswald\\" font-weight=\\"400\\" font-size=\\"99.751\\" font-family=\\"Oswald\\" letter-spacing=\\"0\\" word-spacing=\\"0\\" fill=\\"#313131\\" stroke-width=\\"0.397\\"><path d=\\"M14.18 34.54h5.917c4.3 0 7.58-1.216 9.84-3.648 2.261-2.433 3.391-5.838 3.391-10.216 0-4.865-1.064-8.557-3.191-11.078-2.084-2.521-5.452-3.781-10.107-3.781H6.666L6.63 39.902H0V0h39.994v39.902H14.173z\\"></path><path d=\\"M17.637 29.167c2.216 0 3.923-.243 5.12-.73 1.24-.486 2.127-1.326 2.659-2.52.532-1.195.798-2.92.798-5.175 0-2.742-.2-4.754-.599-6.036-.398-1.283-1.196-2.19-2.393-2.72-1.153-.53-2.992-.796-5.519-.796H14.18v17.977z\\"></path></g><text style=\\"line-height:120%;-inkscape-font-specification:Oswald\\" x=\\"43.138\\" y=\\"136.474\\" font-weight=\\"400\\" font-size=\\"6.063\\" font-family=\\"Oswald\\" letter-spacing=\\"0\\" word-spacing=\\"0\\" fill=\\"#313131\\" stroke-width=\\"0.264\\" transform=\\"matrix(1.00426 0 0 1.00194 -43.706 -87.826)\\"><tspan x=\\"43.138\\" y=\\"136.474\\">Dr. Pogodin Studio</tspan></text></g></svg>"`;
